src/libraries/System.Private.CoreLib/src/System/Environment.OSVersion.OSX.cs

```diff
@@ -7,7 +7,15 @@ public static partial class Environment
     {
         private static OperatingSystem GetOSVersion()
         {
-            Version version = Interop.libobjc.GetOperatingSystemVersion();
+            Version version = GetMacOSVersion();
+
+            if (version.Major == 10 && version.Minor == 16)
+            {
+                // We get "compat" version for 11.0 unless we build with updated SDK.
+                // Hopefully that will be before 11.x comes out
+                // For now, this maps 10.16 to 11.0.
```

Comment:
Is there any issue tracking updating this if that happens (or if it doesn't)?

Reply:
To my knowledge there's no issue tracking this.

cc: @eerhardt

Reply:
Well, 11.x versions are already out so that should not be hard to verify